operator/buffer/memory.go

```diff
@@ -39,7 +39,7 @@ func NewMemoryBufferConfig() *MemoryBufferConfig {
 
 // Build builds a MemoryBufferConfig into a Buffer, loading any entries that were previously unflushed
 // back into memory
-func (c MemoryBufferConfig) Build(context operator.BuildContext, pluginID string) (Buffer, error) {
+func (c MemoryBufferConfig) Build(_ operator.BuildContext, pluginID string) (Buffer, error) {
```

Comment:
It's satisfying and interface where it may be used in other implementations. In the buffer-rework stuff we removed this from being passed into the buffers as nothing used it.